src/dnd/DragArea.jsx

```diff
@@ -112,24 +119,65 @@ class DragArea extends Component {
     return currentContainer ? currentContainer.id : null;
   };
 
+  renderDraggable = ID => (
+
+    <Draggable
+      key={ID}
+      ID={ID}
+      setDraggable={this.setDraggable}
+    >
+      {this.props.renderDraggableByID(ID)}
+    </Draggable>
+
+  );
+
+  renderDroppableContainer = (container) => {
+    const draggables = Object.keys(container.draggables).map(this.renderDraggable);
+    return (
+      <DroppableContainer
+        key={container.ID}
+        ID={container.ID}
+        setDroppable={this.setDroppable}
+      >
+        {this.props.renderDroppableByID(
+          draggables, container)}
+      </DroppableContainer>
+    );
+  };
+
+  renderContent = () => Object.values(this.state.containers).map(this.renderDroppableContainer);
```

Comment:
Redundant

Just use this code inside render

Reply:
Ready